freeipa.spec.in

```diff
@@ -963,6 +963,11 @@ if [ $1 -gt 1 ] ; then
     if [ $restore -ge 2 ]; then
         %{__python3} -c 'from ipaclient.install.client import update_ipa_nssdb; update_ipa_nssdb()' >>/var/log/ipaupgrade.log 2>&1
     fi
+
+    if [ $restore -ge 2 ]; then
+        sed -E 's/^(HostKeyAlgorithms ssh-rsa,ssh-dss)$/# disabled by ipa-client update\n# \1/' /etc/ssh/ssh_config > /etc/ssh/ssh_config.ipanew
```

Comment:
sed has inplace editing feature (`sed -i`), you don't need to move files around. When using `sed -i.orig` you'd also save the old version.

Reply:
Other hooks in the RPM spec file also use this approach. There is probably a good reason to not use in-place op here.

Reply:
No, other hooks are just as outdated. It was never implied to avoid using inplace editing.